src/tree.rs

```diff
@@ -0,0 +1,170 @@
+// FIXME: Remove once we have a binary in place.
+#![allow(dead_code)]
+use std::{fs, path::Path, str::FromStr};
+
+use ethers::types::{Address, H256, U256};
+use zk_evm::aux_structures::LogQuery;
+use zksync_merkle_tree::{Database, MerkleTree, RocksDBWrapper};
+
+use eyre::Result;
+
+use crate::{state::CommitBlockInfoV1, INITAL_STATE_PATH};
+
+pub struct TreeWrapper<'a> {
+    pub tree: MerkleTree<'a, RocksDBWrapper>,
+    // FIXME: How to save this for persistant storage?
+    pub index_to_key: Vec<U256>,
```

Comment:
Could we just write it ~as is? i.e. `<num values: 32bit unsigned>,<value0>, <value1>, <value2>...` - all numbers big endian encoded?